example/ci/pipeline.yaml

```diff
@@ -230,14 +122,33 @@ jobs:
     plan:
       - in_parallel:
           - get: source
-          - get: s3-gitversion
-            passed: [Generate gitversion and put it on S3]
             trigger: true
-            params:
-              unpack: true
+          - get: image-bb-gitversion-tool
+      - task: generate-version
+        image: image-bb-gitversion-tool
+        config:
+          platform: linux
+          inputs:
+            - name: source
+          outputs:
+            - name: gitversion
+          run:
+            path: /bin/bash
+            args:
+              - -exc
+              - |
+                entrypoint.sh
```

Comment:
on purpose? looks a bit strange for me...

Reply:
yes, see https://concourse-ci.org/tasks.html#task-environment

It can be, that it works without the pipe for a one liner, though